src/browser/css/components/MainPage.css

```diff
@@ -11,3 +11,64 @@
 div[id*="-permissionDialog"] {
   max-width: 350px;
 }
+
+.finder {
```

Comment:
I feel `Finder.css` should be created. And only `.finder`'s location in MainPage should be defined here.

Reply:
Done